visualization/app/codeCharta/ui/customConfigs/customConfigList/customConfigItemGroup/customConfigDescription/customConfigTruncateNote.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { CustomConfigTransformNote } from "./CustomConfigTransformNote.pipe"
+
+describe("customConfigTruncateNotePipe", () => {
+	it("should return empty placeholder sting if no string given", () => {
+		expect(new CustomConfigTransformNote().transform("", 0)).toBe("Add note")
+	})
+
+	it("should return a shortened string when given string above limit", () => {
+		expect(new CustomConfigTransformNote().transform("Shorten This", 5)).toBe("Short...")
```

Comment:
Nit: this would be more readable
```suggestion
		expect(new CustomConfigTransformNote().transform("123456", 5)).toBe("Short...")
```